pandas/tests/frame/test_query_eval.py

```diff
@@ -370,27 +361,25 @@ def to_series(mi, level):
                 raise AssertionError("object must be a Series or Index")
 
     def test_raise_on_panel_with_multiindex(self, parser, engine):
-        tm.skip_if_no_ne()
```

Comment:
Minor bug before where this wasn't passing ``engine`` as an argument to the skip_if function, so this test was always being skipped (even for non-``numexpr`` engines)